### PR TITLE
[SPARKC-571] Control resolving of the contact points

### PR DESCRIPTION
**Pull request by alexott**

### CODE REVIEW

File: connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala [hunk 1]
```diff
@@ -240,6 +241,12 @@ object CassandraConnectorConf extends Logging {
     deprecatedSince = "DSE 6.0.0"
   )
 
+  val ResolveContactPoints = ConfigParameter[Boolean](
+    name = "spark.cassandra.connection.resolveContactPoints",
+    section = ReferenceSection,
+    default = true,
+    description = """Should we resolve contact points to IPs on start (true, false)""")
```
Comment (RussellSpitzer) [inline connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala]: We can be more helpful here, Perhaps "Resolves contact points on every connection. Helpful for usage with Kubernetes or other systems with Dynamic Endpoints which may change while the application is running.

**Comment by RussellSpitzer** [comment]
Rebase this onto b3.0 too please

File: connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala (hunk 1 above)
Comment (alexott) [reply connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala]: done

**Comment by alexott** [comment]
rebased onto b3.0 as well

**Comment by RussellSpitzer** [comment]
Changed target to b3.0

**Comment by RussellSpitzer** [comment]
Notice this Travis Please

**Comment by RussellSpitzer** [comment]
I'm going to close and reopen this to trigger Travis

**Comment by RussellSpitzer** [comment]
Hurray it worked

**Pull request merged**